src/main/java/org/opentripplanner/standalone/config/routerequest/RouteRequestConfig.java

```diff
@@ -462,6 +415,60 @@ private static void mapBikePreferences(NodeAdapter c, BikePreferences.Builder bu
             "How bad is it to walk the bicycle up/down a flight of stairs compared to taking a detour."
           )
           .asDouble(dft.stairsReluctance())
+      )
+      .withParking(it ->
+        it
+          .withUnpreferredVehicleParkingTagCost(
+            c
```

Comment:
The first 3 items are identical with the ones below. Can you make them bike/car specific or extract a re-usable method?

Reply:
I think I'll create a reusable method if possible, I'll check.

Reply:
I think I'll do it in a follow up pr because some of the parameter names are not  generic now (they will be in in the future).